### ISSUE TITLE
Warning: Detected code that calls async_forward_entry_setup

### ISSUE THREAD

**Issue by tjorim** [olibos/HomeAssistant-RecycleApp#71]
```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:151
First occurred: 00:28:51 (7 occurrences)
Last logged: 00:30:33

Detected code that calls async_forward_entry_setup for integration recycle_app with title: 7 Gildenstraat, 2470 - Retie and entry_id: c8a3d728c994a46553b6290280963749, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1. Please report this issue.
```

**Comment by olibos** [comment]
Hello @tjorim ,

Good catch!
It seems there is another issue:
```text
This method is deprecated and will stop working in Home Assistant 2025.6.

Instead, await async_forward_entry_setups as it can load
multiple platforms at once and is more efficient since it
does not require a separate import executor job for each platform.
```

I'll patch it on the sametime.

Thank you!

**Comment by tjorim** [comment]
Ah yes, I seemed to have missed below:

```
Detected that custom integration 'recycle_app' calls async_forward_entry_setup for integration, recycle_app with title: 7 Gildenstraat, 2470 - Retie and entry_id: c8a3d728c994a46553b6290280963749, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/recycle_app/__init__.py, line 138: hass.async_create_task(, please create a bug report at https://github.com/olibos/HomeAssistant-RecycleApp/issues
```

**Comment by olibos** [comment]
Hello @tjorim 
A new pre-release is pushed to fix this issue: [v2.5.1-b1](https://github.com/olibos/HomeAssistant-RecycleApp/releases/tag/v2.5.1-b1)

Can you check on your side ?

**Comment by tjorim** [comment]
Just tested it out, all good!
